### PR TITLE
Improve build duration visibility

### PR DESCRIPTION
**Pull request by samcolson4**
Looking at build duration before 

Looking at build duration now 

This is a small UI change, but a crucial one for UX. Where before, the 'build duration' data was hard to parse at a glance, let alone when it has been compressed down a Zoom share screen, changing the text to black allows the user to get information quickly, without ruining the aesthetic of concourse.

### CODE REVIEW

**Comment by taylorsilva** [comment]
errors when running `yarn build`: https://ci.concourse-ci.org/builds/271808721#L61463cbe:10
```
NameError: #000000 is undefined in /Users/tasilva/workspace/concourse/concourse/web/assets/css/production.less on line 228, column 3:
227 .dict-key {
228   #000000
229 }

error Command failed with exit code 1.
```

**Comment by samcolson4** [comment]
> errors when running `yarn build`: https://ci.concourse-ci.org/builds/271808721#L61463cbe:10
> 
> ```
> NameError: #000000 is undefined in /Users/tasilva/workspace/concourse/concourse/web/assets/css/production.less on line 228, column 3:
> 227 .dict-key {
> 228   #000000
> 229 }
> 
> error Command failed with exit code 1.
> ```

This is fixed in the latest commit. Thanks for catching it.

**Comment by taylorsilva** [comment]
@samcolson4 can you sign your commits please? It's quite common to sign your commits for OSS projects.

You can try running `git rebase HEAD~3 --signoff` on your branch and that should add the sign-off line to each commit if your `GIT_AUTHOR_NAME` and `GIT_AUTHOR_EMAIL` are configured. You'll probably have to do a force push after.

If that doesn't work you can try manually editing your three commits to include the sign-off line.

Signing your commits means each commit has your name + email address in this format:
```
Signed-off-by: Your Name <youremail@example.com>
```

You can see what a signed commit looks like here: https://github.com/concourse/concourse/commit/efb041901c18a74a5c6d91df5a5547b7ffe09c2a

Let me know if you have any other questions. Then this will be good to merge.

**Comment by taylorsilva** [comment]
Actually I don't think this change is necessary. We just did a UI overhaul with the colours in #7497 which also darkens the build duration text. That PR will be in the next release, 7.6.0.
The UI for the build page looks like this after that PR:
image

---

Your change also darkens the text in other areas that decreases visibility. Guessing your branch didn't start off the latest on `master`.
image

We really appreciate the PR, but I don't think we need it :)
We had a designer that picked all the new colours, you can see the work they did here: https://github.com/concourse/concourse/issues/6029
